Description: add fdo keywords
 Add play, paind, draw and art keywords to .desktop file

--- tuxpaint-0.9.22.orig/src/tuxpaint.desktop
+++ tuxpaint-0.9.22/src/tuxpaint.desktop
@@ -113,6 +113,7 @@ Type=Application
 Exec=tuxpaint
 Icon=tuxpaint
 Terminal=false
+Keywords=play;paint;draw;art;
 Categories=Education;Art;
 GenericName=Drawing program
 GenericName[ach]=Purugram me goc
--- tuxpaint-0.9.22.orig/src/tuxpaint.desktop.in
+++ tuxpaint-0.9.22/src/tuxpaint.desktop.in
@@ -5,5 +5,6 @@ Exec=tuxpaint
 Icon=tuxpaint
 Terminal=false
 Categories=Education;Art;
+Keywords=play;paint;draw;art;
 _GenericName=Drawing program
 _Comment=A drawing program for children.
